server/athenian/api/models/state/models.py

```diff
@@ -167,3 +167,28 @@ class ReleaseSetting(create_time_mixin(updated_at=True), Base):
     branches = Column(String(1024))
     tags = Column(String(1024))
     match = Column(SmallInteger())
+
+
+class Feature(create_time_mixin(updated_at=True), Base):
+    """Product features."""
+
+    __tablename__ = "features"
+    __table_args__ = (UniqueConstraint("name", "component", name="uc_feature_name_component"),
+                      {"sqlite_autoincrement": True})
+
+    id = Column(Integer(), primary_key=True)
+    name = Column(String(128), nullable=False)
+    component = Column(String(64), nullable=False)
```

Comment:
Would it worth using an enum type? That would also make the column more self-documented.

Reply:
ok